eng/pipelines/templates/jobs/ci.tests.yml

```diff
@@ -86,10 +86,17 @@ jobs:
       value: "PLAYBACK"
     - name: AZURE_LOG_LEVEL
       value: "verbose"
+    - name: Codeql.Enabled 
```

Comment:
@danieljurek is this the right place for these? We only really want this for the build job(s)

Reply:
It would be great to see a pipeline test run in this PR that is enabling the CodeQL job, to see what the results look like (even if that means moving it to different job temporarily).

@danieljurek which pipeline run can I see CodeQL being enabled on, when a change is made here?

Reply:
@weshaggard there are no standalone jobs (build, etc.) hence why we ended up putting it here. I'd love to find a better place for it to live.

Reply:
We build with a variety of tools (visual studio, gcc, clang)... Do we want CodeQL coverage in each of those or should we put it in a specific build?

Reply:
@danieljurek we just need it once. In other pipelines there's a separate Build or Analyze job that's not multiplied by matrix parameters.